spot-contracts/contracts/vaults/RolloverVault.sol

```diff
@@ -75,6 +75,14 @@ contract RolloverVault is
     /// @dev The maximum number of deployed assets that can be held in this vault at any given time.
     uint256 public constant MAX_DEPLOYED_COUNT = 47;
 
+    //-------------------------------------------------------------------------
+    // Storage
+
+    /// @notice The enforced minimum amount of assets to that have to be deployed in each iteration.
```

Comment:
Something like the below? I think it's worthwhile saying how the value is denominated.
```suggestion
    /// @notice Minimum amount of assets that must be deployed for a deploy operation to succeed, denominated in perps.
```